Trackers/iou_pred_tracker/iou_pred.py

```diff
@@ -0,0 +1,222 @@
+import numpy as np
+from ..track import Track
+from scipy.optimize import linear_sum_assignment as linear_assignment
+import math
+
+class iou_pred_track(Track):
+    def __init__(self, id, bbox, hits, miss):
+        self.history = np.array([[0,0,0,0],[0,0,0,0],[0,0,0,0]])
+        Track.__init__(self, id, bbox, hits, miss)
+
+    def iou_predict(self, bbox, history):
```

Comment:
whats the need of those argument which are already present in the class properties?